### DISCUSSION TITLE
How to slow down the carla server running on windows

### DISCUSSION THREAD

**Discussion by barrykidney** [carla-simulator/carla#6571] [Q&A] [answered]
I'm running Carla 0.9.14 on Windows 11 and the server seems to run at about double normal speed. I'm using the manual_control.py to generate training data and I'm finding it difficult to record realistic driving maneuvers with all the vehicles and pedestrians moving at double time. Is there a way to control the speed that the server runs at? I start the server with the .exe file do I need to pass an arg to it? if so what arg and val will reduce the speed. Thanks

**Comment by rhalaly** [comment, marked as answer]
You can use [Fixed time-step](https://carla.readthedocs.io/en/0.9.14/adv_synchrony_timestep/#fixed-time-step) to slow down the simulator

**Comment by barrykidney** [comment]
Thanks #rahalay, I hadn't realized how to use the config file lol, so for anyone else struggling with this, once the server is started just run "python <path_to_carla>/PythonAPI/util/config.py --delta-seconds 0.025"
